build/cmake/functions.cmake

```diff
@@ -342,22 +351,23 @@ function(wx_set_target_properties target_name is_base)
     if(wxBUILD_SHARED)
         string(TOUPPER ${target_name_short} target_name_upper)
         if(target_name_short STREQUAL "mono")
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL)
-        else()
-            target_compile_definitions(${target_name} PRIVATE DLL_EXPORTS WXMAKINGDLL_${target_name_upper})
-        endif()
-        if(NOT target_name_short STREQUAL "base")
-            target_compile_definitions(${target_name} PRIVATE WXUSINGDLL)
+            target_compile_definitions(${target_name} PRIVATE WXMAKINGDLL)
+        elseif(NOT wxTARGET_IS_PLUGIN)
+            target_compile_definitions(${target_name} PRIVATE WXMAKINGDLL_${target_name_upper})
         endif()
+        target_compile_definitions(${target_name} INTERFACE WXUSINGDLL)
+    endif()
+    if(wxTARGET_IS_PLUGIN OR (wxBUILD_SHARED AND NOT target_name_short STREQUAL "base"))
+        target_compile_definitions(${target_name} PRIVATE WXUSINGDLL)
```

Comment:
Same with above.

Reply:
Same as the other comment. This define is conditional to some projects, while `INTERFACE WXUSINGDLL` is for all projects.